src/content/DotLiquid/paket.dependencies

```diff
@@ -0,0 +1,21 @@
+storage: none
+source https://api.nuget.org/v3/index.json
+
+clitool dotnet-xunit 2.3.1
+clitool Microsoft.DotNet.Watcher.Tools 2.0.0
+nuget Microsoft.AspNetCore.Cors >= 2.0.0
+nuget Microsoft.AspNetCore.Hosting >= 2.0.0
+nuget Microsoft.AspNetCore.Diagnostics >= 2.0.0
+nuget Microsoft.AspNetCore.Server.Kestrel >= 2.0.0
+nuget Microsoft.AspNetCore.Server.IISIntegration >= 2.0.0
+nuget Microsoft.AspNetCore.StaticFiles >= 2.0.0
+nuget Microsoft.Extensions.Logging.Console >= 2.0.0
+nuget Microsoft.Extensions.Logging.Debug >= 2.0.0
+nuget Microsoft.AspNetCore.Authentication.Cookies >= 2.0.0
+nuget Microsoft.Extensions.DependencyInjection >= 2.0.0
+nuget Giraffe >= 0.1.0-beta-600
```

Comment:
What does `Giraffe >= 0.1.0-beta-600` exactly mean? I had in the past `0.1.0-beta-*` in the dependencies of the template and when I made breaking changes to the beta package of Giraffe it also broke the template for all users, therefore I decided to set the Giraffe package to a fixed version in the template to guarantee that the template will always work regardless of what else has happened in the Giraffe world. A user can always upgrade to the latest Giraffe by manually modifying the `.fsproj` after the `dotnet new giraffe` command. Also I try to push a new version of the template every time there's a new version of Giraffe (hasn't happened for beta-700 yet though) so I hope that having a fixed version is not such a big deal and it feels more secure to me. What do you think?

Reply:
If `Giraffe >= 0.1.0-beta-60*` then I'm happy to leave it like it is, just want to double check if that is the case.

Reply:
To be honest I would need to double check this. I only just started playing with Paket. Maybe @forki could give me some guidance on making it equivalent. Else I will take a look when I wake up from a Xmas food coma :)

Reply:
In paket there is a difference between lock and dependencies file. You will usually just say "nuget Giraffe" in the dependencies file and let paket figure it out. It the commits that to the lock file and the version is fixed for further restore. So it's never floating like in nuget. And still you don't have to manually pin things.

Reply:
In other words: please remove all version pins from the dependencies file.

Reply:
And I mean all versions - not only the pins on Giraffe.

Reply:
@forki  that is a lot of versions to remove :) haha. Ok thanks for the guidance. Will sort that out.

Reply:
Yes, but that's the idea of paket: only pin things manually if you really need to. In contrast to nuget you have the lock file which pins everything for restore.

Reply:
Ok got it, so after removing all pinned versions we should also include the paket.lock file in the template?